### DISCUSSION TITLE
Minio Docker LDAP self-signed

### DISCUSSION THREAD

**Discussion by GratienJAUFFRET** [minio/minio#16727] [Q&A] [answered]
Hello everyone,

I've read [this](https://github.com/minio/minio/discussions/13703) and the related documentation, 
but it seems like the Docker config documentation is lacking.
I added --certs-dir to the container command and provided a volume that points to the container path.
I added the LDAP cert both in der & pem formats to the minio created CAs dir (rw-r--r--), but I still get `x509: certificate signed by unknown authority`

Am I missing smthg ? Did I do smthg wrong ?

Thanks in advance for your answers

**Comment by vadmeste** [comment, marked as answer]
@GratienJAUFFRET what is the output of `openssl s_client -quiet -connect ldap-server-address:port -CAfile /path/to/your/CA-file`

**Reply by GratienJAUFFRET** [reply]
Thank you for your answer.
The certificate that was given to me was not the correct one.
Thanks for pointing me out towards openssl validation.
Now everything is working just fine. 💯
